### ISSUE TITLE
Cannot invoke JSCS on Windows

### ISSUE THREAD

**Issue by brzpegasus** [tomchentw/mocha-jscs#3]
The way JSCS is [invoked](https://github.com/dockyard/ember-suave/blob/v0.1.6/lib/rules/require-enhanced-object-literals.js#L7) is not Windows-friendly. Ultimately, it is because Windows doesn't understand the shebang line in [`jscs/bin/jscs`](https://github.com/jscs-dev/node-jscs/blob/v1.12.0/bin/jscs#L1):

``` javascript
#!/usr/bin/env node
var cli = require('../lib/cli');
// ...
```

This causes all JSCS tests to fail with some non-useful, "Uncaught" error. The simplest thing to do might be to just invoke node and passing the path to the JSCS file.

**Comment by brzpegasus** [comment]
Woops, should have checked open PRs. This is a dupe of #2.

**Comment by tomchentw** [comment]
Released [v1.1.0](https://github.com/tomchentw/mocha-jscs/releases/tag/v1.1.0).
